src/app/mesh/qgsmeshcalculatordialog.cpp

```diff
@@ -248,11 +289,11 @@ void QgsMeshCalculatorDialog::updateInfoMessage()
   bool expressionValid = result == QgsMeshCalculator::Success;
 
   // selected driver is appropriate
-  bool driverValid = false;
+  bool driverValid = mOutputOnMemoryRadioButton->isChecked();
```

Comment:
maybe create local variable for readability? 

bool useInMemory = mOutputOnFileRadioButton->isChecked();